### ISSUE TITLE
poor detection accuracy

### ISSUE THREAD

**Issue by troylhy1991** [NVlabs/OmniDrive#17]
the planning evaluation is close to what the paper reported after training for 6 epochs. however, the detection scores are very low, is this expected?  ;(

**Comment by exiawsh** [comment]
Yes. Detection requires long time to converge.

**Comment by troylhy1991** [comment]
many thx! turns out i was using default lr 1e-4 while my batchsize was only half.    but it kind of introduces another confusing part to me: the planning sub task might be too easy to overfit ...... ;(
